skimage/transform/histogram_matching.py

```diff
@@ -6,9 +6,9 @@ def _match_cumulative_cdf(source, template):
     Return modified source array so that the cumulative density function of
     its values matches the cumulative density function of the template.
     """
-    src_values, src_unique_indices, src_counts = np.unique(source.ravel(),
-                                                           return_inverse=True,
-                                                           return_counts=True)
+    _, src_unique_indices, src_counts = np.unique(source.ravel(),
```

Comment:
I'm -1 on these 3 changed lines. These kinda feel like "flavour of the week" kinda of changes, which are only side steps, and not clear "improvements".

Reply:
sorry, I do not understand "flavour of the week"

Reply:
flavour of the week is a statement I created that means that the motivations for this modification change every week depending. In a similar fashion that a desert store will have a new flavour every week.

These kinds of changes make the git history very hard to read, and don't really add too much value. I could agree that `src_values` reminds us what the `_` refers to, in this not so known function.

Reply:
by my understanding, this change is recommended from codacy.com and codefactor.io and it is the same kind of change as yours moving `not` or removing `results`, https://github.com/scikit-image/scikit-image/pull/3224